cmd/crc/cmd/daemon_windows.go

```diff
@@ -2,8 +2,21 @@ package cmd
 
 import (
 	"net"
+
+	"github.com/Microsoft/go-winio"
+	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/logging"
 )
 
 func httpListener() (net.Listener, error) {
-	return nil, nil
+	ln, err := winio.ListenPipe(constants.DaemonHTTPSocketPath, &winio.PipeConfig{
```

Comment:
`constants.DaemonHTTPSocketPath` This name is quiet cryptic and not even true in this case;

Reply:
I added a new variable for named pipe. It should be better now.